src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -285,7 +301,9 @@ trait Qualif {
     }
 }
 
-// Constant containing interior mutability (UnsafeCell).
+/// Constant containing interior mutability (UnsafeCell).
+/// This must be ruled out to make sure that evaluating the constant at compile-time
+/// and run-time would produce the same result.
```

Comment:
well... more ensuring that at runtime you don't see different values for the same constant depending on how early/late you are querying during execution